doc/contributor-how-to/how-to-dev-plugin.md

```diff
@@ -0,0 +1,64 @@
+# How to install QGreenland Custom as a developer
+
+QGreenland Custom is a QGIS plugin for downloading a custom set of data, including data which 
+is not part of the QGreenland Core zip package. This how-to guide walks through how to install the plugin as a developer.
+
+First, clone the QGreenland Custom repo from Faunalia:
+
+```
+git clone https://github.com/faunalia/qgreenland-plugin.git
+```
+
+Use the QGIS GUI to locate the QGreenland Custom directory by going to
+Settings -> User Profiles -> Open Active Profile Folder.
```

Comment:
Do we have a pre-existing style for this type of instruction? Should we enclose it in backticks, or maybe a block-quote to make it clearer? Whatever we do, we should be consistent!

e.g.?

>Settings -> User Profiles -> Open Active Profile Folder.

Reply:
What do you mean by pre-existing style?

Reply:
got it, I'll go with `>`

Reply:
I think it's great to use a consistent symbol, but I'm more thinking about: do we have a way to offset these "menu paths"? Without any special highlighting, it's hard to tell which part of a sentence is a Menu Path. We have options like

```
menu -> path -> to -> thing
```

> menu -> path -> to -> thing

Or even *menu -> path -> to -> thing* could work?

Reply:
Once we've made this decision, let's also commit a "documentation style guide" as a reference page so we can reference it later.

Reply:
What do you think of this for the style guide:

>>>* Any reference to a QGIS user interface element will be bolded. e.g.: "Navigate to **Settings -> User Profiles -> Open Active Profile Folder** to get the directory path." or "Use the **Layers Panel** to change the order of the layers.

* Menu paths are delimited by `->`, e.g.: **Settings -> User Profiles -> Open Active Profile Folder**

Reply:
Would you be willing to add that style guide in this pull request?

Reply:
sure I can do that